### PR TITLE
updated opencti-RT-env as per latest structure

### PR DESCRIPTION
**Pull request by NeerajK007**
updated opencti-RT-env as per latest structure with ansible role

### CODE REVIEW

**Review by pramit-d — CHANGES_REQUESTED** [review]
@NeerajK007 env installation failing.

Screenshot from 2024-03-20 16-24-14

**Review by asa1997 — CHANGES_REQUESTED** [review]
@NeerajK007 Please update the env scripts to the latest format.

File: newrelic/0.0.1/besman-newrelic-java-agent-BT-env-config.yaml
Comment (asa1997) [inline newrelic/0.0.1/besman-newrelic-java-agent-BT-env-config.yaml:1]: @NeerajK007 This file is still in the old format

File: newrelic/0.0.1/besman-newrelic-java-agent-BT-env.sh
Comment (asa1997) [inline newrelic/0.0.1/besman-newrelic-java-agent-BT-env.sh:1]: @NeerajK007 Please update this file

File: opencti/0.0.1/besman-opencti-RT-env.sh [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+    else
+        echo "criticality_score is already available"
+    fi
+
+    ## setup snyk using yarn
+
+    echo -e "\nopencti RT env installation is complete"
+}
+
+function __besman_uninstall {
```
Comment (asa1997) [inline opencti/0.0.1/besman-opencti-RT-env.sh:160]: You should uninstall/update/reset/validate everything that was installed.

**Review by asa1997 — CHANGES_REQUESTED** [review]
@NeerajK007 Please address the comments

File: opencti/0.0.1/besman-opencti-RT-env.sh (hunk 1 above)
Comment (NeerajK007) [reply opencti/0.0.1/besman-opencti-RT-env.sh:159]: Hi Arun,
Uninstall and validate functions are done.
I don't know what will go in reset and update function ??

**Comment by asa1997** [comment]
@NeerajK007 Please resolve the conflicts

**Review by asa1997 — APPROVED** [review]
@NeerajK007 LGTM

**Pull request merged**